### PR TITLE
Fix chat spacings

### LINKED ISSUE #19013: Fix chat messages spacings
# Problem
Chat spacing issues from [design review](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=8065%3A58901&mode=design&t=gv0MnVcExvcN5Emp-1)

1. Fix spacing between system messages
2. Fix spacing between date and mute/pinned buttons
3. Fix spacing between last message and shell button

### PR DESCRIPTION
**Pull request by clauxx**
fixes #19013 

### Summary

Fix the spacing issues from the chat screen

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

status: ready

### CODE REVIEW

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs [hunk 1]
```diff
@@ -268,25 +268,31 @@
 (defn message
   [{:keys [pinned-by mentioned content-type last-in-group? deleted? deleted-for-me?]
     :as   message-data} {:keys [in-pinned-view?] :as context} keyboard-shown?]
-  [rn/view
-   {:style               (style/message-container in-pinned-view? pinned-by mentioned last-in-group?)
-    :accessibility-label :chat-item}
-   (cond
-     (system-message? content-type)
-     [system-message-content message-data]
+  (let [system-message? (get-system-message? content-type)]
```
Comment (flexsurfer) [inline src/status_im/contexts/chat/messenger/messages/content/view.cljs]: `(check-fi-system-message content-type)` idk might be better.

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/contexts/chat/messenger/messages/content/view.cljs (hunk 1 above)
Comment (clauxx) [reply src/status_im/contexts/chat/messenger/messages/content/view.cljs]: done

**Comment by mariia-skrypnyk** [comment]
Hi @clauxx ! 👋

Please, resolve the conflicts and rebase your PR. 
Thanks!

**Comment by clauxx** [comment]
@mariia-skrypnyk done :+1:

**Comment by flexsurfer** [comment]
hey @clauxx all build are failed, could you pls fix and then move to e2e

**Comment by mariia-skrypnyk** [comment]
Hi @clauxx ! Thanks for rebase.

**ISSUE 1 "Fix spacing between last message and shell button" has incorrect space:**

**Design:** 
Screenshot 2024-03-18 at 14 14 42
https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=1966-150131&mode=design&t=O2C094q2kW12Yf2X-0

**Now:**
Screenshot 2024-03-18 at 14 16 08

**Comment by clauxx** [comment]
@mariia-skrypnyk it looks fine compared to [this](https://www.figma.com/file/wA8Epdki2OWa8Vr067PCNQ/Composer-for-Mobile?type=design&node-id=3101-347945&mode=design&t=t2H9tsN2WWHQQEe2-4). We should compare this spacing (last message to shell button) with designs in which there are scrollable messages (left image) and they're scrolled all the way down. On the right we can't really tell what's the spacing cause there's not enough messages to fill the screen.

**Comment by mariia-skrypnyk** [comment]
Thanks @clauxx .
Everything is clear now. 
And I have no issues more so I move your PR to @Francesca-G review.
Great job!

**Review by Francesca-G — COMMENTED** [review]
Looks better 👍 
We still have some spacing issues as commented [here](https://www.figma.com/file/1IB0ZZ08ODzuoWmixDSzm7/Review-Messages-%2F-Mobile?type=design&node-id=122%3A12875&mode=design&t=X2tWs9oSPmaRxhXV-1)

**Comment by clauxx** [comment]
@Francesca-G fixed a few other spacing issues as well. Here's an overlap of the implementation over the design (used iPhone 11 Pro cause i didn't find iPhone 13 designs for the chat).

Screenshot 2024-03-19 at 16 52 05

**Pull request merged**